test/template-tests/testcases/deploymentTemplate/Resources-Should-Have-Location.test.ps1

```diff
@@ -0,0 +1,17 @@
+﻿param(

```

Comment:
Can you run this on the repo and see what we're catching as failures... i.e. do we ever see a resource that does not have a location property and doesn't need one?  We may need to expand the test to cover that scenario...

Reply:
These are the resources types that currently have failures:


Count Name                                      
----- ----                                      
    5 Microsoft.Storage/storageAccounts         
    5 microsoft.insights/components             
    4 Microsoft.DataFactory/datafactories       
    3 Microsoft.OperationalInsights/workspaces  
    3 Microsoft.Network/virtualNetworks         
    1 Microsoft.DataLakeStore/accounts          
    1 Microsoft.ContainerService/managedClusters
    1 Microsoft.Media/mediaServices

Reply:
can you share the templates/files that are throwing - want to see if they are really valid...